js/ui/selection/selection.strategy.standard.js

```diff
@@ -369,16 +397,25 @@ export default SelectionStrategy.inherit({
         this._updateRemovedItemKeys(keys, oldSelectedKeys, oldSelectedItems);
     },
 
-    isItemDataSelected: function(itemData) {
+    isItemDataSelected: function(itemData, checkPending) {
         const key = this.options.keyOf(itemData);
-        return this.isItemKeySelected(key);
+        return this.isItemKeySelected(key, checkPending);
     },
 
-    isItemKeySelected: function(key) {
-        const keyHash = this._getKeyHash(key);
-        const index = this._indexOfSelectedItemKey(keyHash);
+    isItemKeySelected: function(key, options) {
```

Comment:
Why are the arguments inconsistent? In the `isItemDataSelected` method second arg is a Boolean, in the `isItemKeySelected` method it is an object